ckanext/qgov/common/test_mime_type_validation.py

```diff
@@ -5,8 +5,9 @@
 
```

Comment:
A+ job on making this test discrete without needing the rest of our plugin. Top Notch work @ThrawnCA